test/unit/repositories/test_image.py

```diff
@@ -88,6 +89,108 @@ def test_create(self):
         self.check_create_success()
 
 
+class GetDSL(ImageRepoDSL):
+    """Base class for `get` tests."""
+
+    _obtained_image_id: str
+    _expected_image_out: ImageOut
+    _obtained_image_out: ImageOut
+    _get_exception: pytest.ExceptionInfo
+
+    def mock_get(self, image_id: str, image_in_data: dict) -> None:
+        """
+        Mocks database methods appropriately to test the `get` repo method.
+
+        :param image_id: ID of the image to obtain.
+        :param image_in_data: Dictionary containing the image data as would be required for an
+            `ImageIn` database model (i.e. no created and modified times required).
+        """
+        if image_in_data:
+            image_in_data["id"] = image_id
+        self._expected_image_out = ImageOut(**ImageIn(**image_in_data).model_dump()) if image_in_data else None
+
+        RepositoryTestHelpers.mock_find_one(
+            self.images_collection, self._expected_image_out.model_dump() if self._expected_image_out else None
+        )
+
+    def call_get(self, image_id: str) -> None:
+        """Calls the `ImageRepo` `get` method.
+
+        :param image_id: The ID of the image to obtain.
+        """
+        self._obtained_image_id = image_id
+        self._obtained_image_out = self.image_repository.get(image_id=image_id, session=self.mock_session)
+
+    def call_get_expecting_error(self, image_id: str, error_type: type[BaseException]) -> None:
+        """
+        Calls the `ImageRepo` `get` method with the appropriate data from a prior call to `mock_get`
+        while expecting an error to be raised.
+
+        :param image_id: ID of the image to be obtained.
+        :param error_type: Expected exception to be raised.
+        """
+        self._obtained_image_id = image_id
+        with pytest.raises(error_type) as exc:
+            self.image_repository.get(image_id, session=self.mock_session)
+        self._get_exception = exc
+
+    def check_get_success(self) -> None:
+        """Checks that a prior call to `call_get` worked as expected."""
+
+        self.images_collection.find_one.assert_called_once_with(
+            {"_id": ObjectId(self._obtained_image_id)}, session=self.mock_session
+        )
+        assert self._obtained_image_out == self._expected_image_out
+
+    def check_get_failed_with_exception(self, image_id: str, assert_find: bool) -> None:
```

Comment:
```suggestion
    def check_get_failed_with_exception(self, assert_find: bool) -> None:
```
Can you just use `_obtained_image_id` here?